package com.revature.schoolDatabase.util.exceptions;

public class DataSourceException extends RuntimeException {
    public DataSourceException(String s, Throwable cause) {
        super(s, cause);
    }
}
